pkg/cmd/space/delete/delete.go

```diff
@@ -77,11 +80,11 @@ func deleteRun(f factory.Factory, cmd *cobra.Command) error {
 	}
 
 	return question.DeleteWithConfirmation(f.Ask, "space", itemToDelete.Name, itemToDelete.ID, func() error {
-		return delete(client, itemToDelete)
+		return stopQueueAndDelete(octopus, itemToDelete)
 	})
 }
 
-func delete(client *client.Client, spaceToDelete *spaces.Space) error {
+func stopQueueAndDelete(client *client.Client, spaceToDelete *spaces.Space) error {
```

Comment:
This is going to need a rebase now that we merged the other PR [#169](https://github.com/OctopusDeploy/cli/pull/169/files)